src/api/java/appeng/api/definitions/IBlocks.java

```diff
@@ -99,19 +99,19 @@
 	/*
 	 * misc
 	 */
-	ITileDefinition grindStone();
+	ITileDefinition grindstone();
 
 	ITileDefinition crankHandle();
 
 	ITileDefinition inscriber();
 
-	ITileDefinition wireless();
+	ITileDefinition wirelessAp();
```

Comment:
It could be named `wirelessAccessPoint`, either minecraft or IDEs have autocomplete and it is not that we need to save every bit to put it on punch cards.